docs/source/api/queries.rst

```diff
@@ -1074,9 +1084,15 @@ Possible Stateful Validation Errors
     :header: "Code", "Error Name", "Description", "How to solve"
 
     "1", "Could not get block streaming", "Internal error happened", "Try again or contact developers"
-    "2", "No such permissions", "Query's creator does not have any of the permissions to get blocks", "Grant the necessary permission: individual, global or domain one"
+    "2", "No such permissions", "Query's creator does not have any of the permissions to get blocks", "Grant `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission"
     "3", "Invalid signatures", "Signatures of this query did not pass validation", "Add more signatures and make sure query's signatures are a subset of account's signatories"
 
+.. note::
+    `BlockErrorResponse` contains only `message` field.
+    In case of stateful validation error it will be "stateful invalid".
+    `GetBlock <#get-block>`__ requires same `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission.
+    Therefore, it can be used with `height = 1` (first block is always present) to get verbose error.
```

Comment:
Not clear enough for me. What do you mean by *verbose error*? If it is about checking one's permissions - please write that, otherwise please explain the case you mean.